frontend/src/app/app.module.ts

```diff
@@ -11,13 +11,18 @@ import { WindowService, WindowFactory } from './window.service';
 import { HomeComponent } from './home/home.component';
 import { FooterComponent } from './footer/footer.component';
 import { HeaderComponent } from './header/header.component';
+import { AboutUsComponent } from './static/about-us.component';
 
 const appRoutes: Routes = [
   // {
   //   path: '',
   //   component: HomeComponent
   // },
   {
+    path: 'about-us',
```

Comment:
This page on old Django was available under `o-nas` url. Please keep it that way.

Reply:
It was under `about-us`.